WooCommerce/Classes/Analytics/WooAnalyticsStat.swift

```diff
@@ -82,6 +82,12 @@ public enum WooAnalyticsStat: String {
     case loginInvalidEmailScreenViewed = "login_invalid_email_screen_viewed"
     case whatIsWPComOnInvalidEmailScreenTapped = "what_is_wordpress_com_on_invalid_email_screen"
     case createAccountOnInvalidEmailScreenTapped = "create_account_on_invalid_email_screen"
+    case loginInsufficientRole = "login_insufficent_role"
```

Comment:
Hicham reported this typo. p1674132650487609/1673873024.592979-slack-C046HDZL87J
```suggestion
    case loginInsufficientRole = "login_insufficient_role"
```

Reply:
Fixed in 6c5d14429a7a0377492104cd4bd1c450a088d34f.